src/main/resources/macchiato/app/views/contact/contact.html

```diff
@@ -1,6 +1,20 @@
 <!-- contact.html -->
 <div class="jumbotron text-center">
-	<h1>Contact Page</h1>
-
+	<h1>Contact Us</h1>
+   <!--  <h2>Feel free to contact us. Please select the option the suits you the best.</h2> -->
 	<p>{{ message }}</p>
+</div>
+<div class="contactType">
+	<div class="call">
+		<div class="contact-icon"><i class="fa fa-mobile fa-2x"></i></div>
+		<div class="contact-text"><p>Call us at<br>123.456.7890</p></div>
```

Comment:
Instead of hard-coding the message directly into the HTML, Can you move this message to js file? like {{ message }} above?